### ISSUE TITLE
Add angrok tool with docker

### ISSUE THREAD

**Issue by YevheniiVolosiuk** [YevheniiVolosiuk/spin-template-wordpress-basic#4]
Make it possible to have staging environment live URL links from [ngrok](https://ngrok.com/docs/using-ngrok-with/wordpress/) for a WordPress.

**Comment by YevheniiVolosiuk** [comment]
Can I use [Server Side Up CA](https://serversideup.net/ca/) for the [ngrock](https://ngrok.com/) stage/dev live URL?
@jaydrogers

**Comment by jaydrogers** [comment]
Yup, I think it will work just fine. Ngrok might use their own proxy inbetween and trust it anyways.
